cmd/uplink/cmd/setup_unexported_test.go

```diff
@@ -0,0 +1,69 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package cmd
+
+import (
+	"io/ioutil"
+	"math/rand"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"storj.io/storj/internal/testcontext"
+	"storj.io/storj/pkg/storj"
+)
+
+func TestSaveEncryptionKey(t *testing.T) {
```

Comment:
The test is here because the function is implemented in _setup.go_ otherwise it would go there; my rationale why it's in _setup.go_ is as a response of your other commend.

Regarding to have an integration test between both, I'm that much sold on it, because the integration test should be more between this _save_ function and the _load_ one present in the `uplink` package, however it looks odd to me to have the integration test of 2 different package in one of them because I'd expect to have it in third one for such purpose, but

1. _save_ isn't exported
2. if it would possible, I'd be in a doubt in which one, but it doesn't matter because 1 forbid to have it in a third package.

Nonetheless, I didn't contemplated to have such test and I think that's worthwhile to have it despite of the odds mentioned above because we don't have that much test on the CLI, hence I'm going to add it.

Reply:
I saw the integration test you added and it looks good to me